src/components/reusable/ConfirmDetails.tsx

```diff
@@ -147,31 +147,33 @@ const ConfirmDetails = ({ isApproved }: { isApproved: boolean }) => {
         </div>
       )}
       <div className='detail-item amount'>
-        <span className='label'>Amount:</span>
+        <span className='label'>
+          Transaction
+          <br />
+          Details:
+        </span>
         <span className='amount-container'>
-          <div className='coin-details'>
-            <SourceCoinIcon />
-            <p>
-              {amountToShow} {sourceCurrency}
-            </p>
-          </div>
-          {sourceCurrency !== targetCurrency && (
+          <div className='amount-details'>
+            <span>Transfer amount</span>
             <div className='coin-details'>
-              → <TargetCoinIcon /> {targetCurrency}
+              <p>
+              {formatterFloat.format(parseFloat(amountToShow)-totalFeeUsd)} {sourceCurrency}
+              </p>
             </div>
-          )}
+            {sourceCurrency !== targetCurrency && (
+              <div className='coin-details'>→ {targetCurrency}</div>
+            )}
+          </div>
```

Comment:
The spacing here makes the alignment of the "Transferring..." text  look off. It's centered within it's area but not centered relative to the progress bar or the status box.

![image](https://github.com/user-attachments/assets/55c491db-ccd1-4158-a7f9-57517e01c033)

Reply:
I don't see the same, in my case it looks perfect D: